build/windows/package_msi.ps1

```diff
@@ -0,0 +1,64 @@
+<#
+    .SYNOPSIS
+        This script creates the win .MSI
+#>
+param (
+    # Target architecture: amd64 (default) or 386
+    [string]$integration="none",
+    [ValidateSet("amd64", "386")]
+    [string]$arch="amd64",
+    [string]$tag="v0.0.0",
+    [string]$pfx_certificate_base64="none",

```

Comment:
`pfx_certificate_base64` Isn't used in the script. Was the plan to use it for importing the cert?